### ISSUE TITLE
Trap focus in popup modals

### ISSUE THREAD

**Issue by teshsopher** [tl-its-umich-edu/remote-office-hours-queue#162]
For screen readers, when a popup occurs, focus should be trapped in the modals.

Popup modals where this is an issue:
- Delete queue
- Leave queue
- Delete host

**Comment by jlost** [comment]
@teshsopher I'm having trouble reproducing this. In Chrome and Firefox, when I open a modal and press TAB, first Cancel is focused, then next OK is focused, and then it loops back around to Cancel. This goes for Join Info, Delete Queue, Delete Host. Leave Queue doesn't produce a modal. Maybe this was fixed somewhere along the way?

**Comment by teshsopher** [comment]
I can't reproduce this either, so I think you're right!
